testgres/utils.py

```diff
@@ -68,6 +68,27 @@ def generate_app_name():
     return 'testgres-{}'.format(str(uuid.uuid4()))
 
 
+def generate_system_id():
+    """
+    Generate a new 64-bit unique system identifier for node.
+    """
+
+    import datetime
+    import struct
+
+    date = datetime.datetime.now()
+    secs = int(date.timestamp())
+    usecs = date.microsecond
+
+    system_id = 0
+    system_id |= (secs << 32)
+    system_id |= (usecs << 12)
+    system_id |= (os.getpid() & 0xFFF)
+
+    # pack ULL in native byte order
+    return struct.pack('=Q', system_id)
```

Comment:
This will probably bail out with `struct.error: argument out of range` after 19 January 2038, but for now it is fine :)

Reply:
@arssher Why? We have 32 bits for seconds, and that's ~136 years.

Reply:
Ah, =Q is unsigned. Yes, then we have 68 years more.